.github/workflows/ci.yml

```diff
@@ -114,6 +114,30 @@ jobs:
   # Building #
   ############
 
+  appcast-edge:
+    name: appcast (edge)
+    runs-on: ubuntu-latest
+    env:
+      NOTES: ${{ secrets.APPCAST_NOTES }}
+      ARTIFACTS: ${{ secrets.ARTIFACTS_MAIN }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Unshallow the repository in order for `git describe` to work.
+          fetch-depth: 0
+
+      - run: |
+          APPCAST=$(make appcast.notes link=$ARTIFACTS notes=$NOTES)
+          echo "$APPCAST" > appcast.xml
```

Comment:
@SleepySquash why not directly?
```bash
make appcast.notes link=$ARTIFACTS notes=$NOTES > appcast.xml
```

Reply:
@tyranron, `> appcast.xml` makes the `git describe --dirty` to return `-dirty` always.